core/src/test/scala/matryoshka/spec.scala

```diff
@@ -455,22 +467,28 @@ class FixplateSpecs extends Specification with ScalaCheck with ScalazMatchers {
 
     "cata" should {
       "evaluate simple expr" in {
-        val v = mul(num(1), mul(num(2), num(3)))
-        v.cata(eval) must equal(6)
-        v.convertTo[Mu].cata(eval) must equal(6)
-        v.convertTo[Nu].cata(eval) must equal(6)
+        testRec(
+          mul(num(1), mul(num(2), num(3))),
+          new RecRunner[Exp, Int] {
+            def run[T[_[_]]: Recursive] = _.cata(eval) must equal(6)
+          })
```

Comment:
Yeah, that doesn’t work, unfortunately. I can do both or just the reader (and DRY, so …).